src/main/resources/org/thoughtslive/jenkins/plugins/hubot/config/HubotJobProperty/config.jelly

```diff
@@ -1,11 +1,11 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:f="/lib/form" xmlns:j="jelly:core">
   <f:section title="Hubot">
-    <f:entry field="enableNotifications" title="Enable Notifications">
-      <f:checkbox/>
+    <f:entry field="enableNotifications">
+      <f:checkbox title="Enable Notifications"/>
     </f:entry>
     <f:entry field="siteNames" title="Site Name">
       <f:select/>
     </f:entry>
   </f:section>
-</j:jelly>
```

Comment:
Can you leave these new lines at the of these files please.

Reply:
That message is actually saying you don't have new lines at the end of your files now and that this change is putting them in.

I can with significant effort avoid adding those end-of-line at end-of-file markers, but each editor will try to put them in, so it's a losing battle and a waste of effort.

Reply:
I want to add those new lines those got removed with this change.

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file